### DISCUSSION TITLE
mutate in Logstash on Manger Node

### DISCUSSION THREAD

**Discussion by 0neNeutrino** [Security-Onion-Solutions/securityonion#9048] [Unsupported Versions] [answered]
Hello,

I am very new to both ELK and Security Onion. I am having a difficult time understanding how to add a mutate filter to Logstash on the manager node of a cluster. I have google'd and read over the documentation and have come up short on a solution. The few things I have tried caused ingest to stop.

**Comment by dougburks** [comment, marked as answer]
We moved all of our default parsing from Logstash to Elasticsearch ingest parsing. For more information, please see:
https://docs.securityonion.net/en/2.3/elasticsearch.html#parsing

**Reply by 0neNeutrino** [reply]
I did see that in the documentation, but we are using multiple search nodes and winlogbeat. This line is responsible for my question: 
"WinLogbeat: Winlogbeat [Windows Endpoint]–> Logstash [Manager] –> Redis [Manager]." 

It's from the Ingest page, but to be honest, I have no idea which section on the Ingest page matches my setup, and am just starting to learn both ELK and SO. I really appreciate the help.

**Reply by dougburks** [reply]
Right, that snippet comes from https://docs.securityonion.net/en/2.3/ingest.html#manager-separate-search-nodes. For a manager with search nodes, logs come in and are stored temporarily in Redis. Then the search nodes pull logs from Redis and parse them using Elasticsearch ingest parsing. You can see that further down that page in the Search section at https://docs.securityonion.net/en/2.3/ingest.html#search.

**Reply by 0neNeutrino** [reply]
Awesome, thanks!
